src/json-patch-duplex.js

```diff
@@ -91,11 +92,13 @@ var jsonpatch;
             return removed;
         },
         move: function (obj, key, document) {
-            var originalValue = getValueByPointer(document, this.path);
-            var newValue = getValueByPointer(document, this.from);
-            applyOperation(document, { op: "remove", path: this.from });
-            applyOperation(document, { op: "add", path: this.path, value: newValue });
-            return originalValue;
+            var overwrittenValue = getValueByPointer(document, this.path);
+            if (overwrittenValue) {
+                overwrittenValue = deepClone(overwrittenValue);
```

Comment:
Ok, fair point. 
However, I'm not 100% sure if it's very useful, but it could be expensive for large object.

Could you add short code comment, jsdoc, or readme node why it's needed. Woe could discuss it later if we would like to improve performance.